server/src/main/java/org/apache/druid/metadata/SQLMetadataStorageActionHandler.java

```diff
@@ -82,7 +83,8 @@ public SQLMetadataStorageActionHandler(
   )
   {
     this.connector = connector;
-    this.jsonMapper = jsonMapper;
+    //noinspection UnnecessaryFullyQualifiedName
+    this.jsonMapper = new DefaultObjectMapper((DefaultObjectMapper) jsonMapper).addMixIn(org.apache.druid.metadata.PasswordProvider.class, org.apache.druid.metadata.PasswordProviderRedactionMixIn.class);
```

Comment:
How about fixing the constructor of `DefaultObjectMapper` to accept `ObjectMapper` instead of `DefaultObjectMapper`. I think it's better because `jsonMapper` doesn't have to be `DefaultObjectMapper`.

Reply:
@jihoonson Oh yes, that makes sense. After checking out the spec for ObjectMapper I think using copy() is the correct way to go with this. It is in the latest commit, let me know what you think about that pattern for this